docs/source/usage/plugins/radiation.rst

```diff
@@ -306,8 +306,177 @@ Command line flag                        Output description
                                          These are for restart purposes and for more complex data analysis.
 ======================================== ========================================================================================================================
 
-Analysing tools
-^^^^^^^^^^^^^^^^
+
+Text-based output
+"""""""""""""""""
+
+The text-based output of ``lastRadiation`` and ``totalRadiation`` contains the intensity values in SI-units :math:`[Js]`. Intensity values for different frequencies are separated by spaces, while newlines separate values for different observation directions. 
+
+
+In order to read and plot the text-based radiation data, a python script as follows could be used:
+
+.. code:: python
+
+    import numpy as np
+    import matplotlib.pyplot as plt
+    from matplotlib.colors import LogNorm
+
+    # frequency definition:
+    # as defined in the 'radiation.param' file:
+    N_omega = 1024
+    omega_min = 0.0 # [1/s]
+    omega_max = 5.8869e17 # [1/s]
+    omega = np.linspace(omega_min, omega_max, N_omega)
+
+    # observation angle definition:
+    # as defined in the 'radiation.param' file:
+    N_observer = 128
+    # as defined in the 'radiationObserver.param' file:
+    theta_min = -1.5 # [rad/gamma]
```

Comment:
It is a little bit unintuitive for me, that the unit of the angle is [rad/gamma] here, but just [rad] in `radiationObserver.param`. I guess it makes sense for the readability of the plot though. And having the angle and the Lorentzfactor as two separate variables can just make is more complicated for the user..

Edit: In the hdf5 script, the user has to enter gamma separately

Reply:
Great input :+1: - I will fix it.